### DISCUSSION TITLE
Need help create framework web to python

### DISCUSSION THREAD

**Discussion by FelipeKatao** [community/community#21381] [Other Feature Feedback, Questions, & Ideas]
<p>Hello guys </p>
<p>Repository link:  <a href="https://github.com/FelipeKatao/Py.Html" title="Project Need help!" rel="nofollow noopener">Py.Html Project</a></p>
<p>I am developing a python framework where the goal is to create a complete website, with systems of Css and Js implemented, through a few lines in python generate an HTML, CSS and JS code where the end user only has to modify what they want, but the shape and layout the Py.html already creates.</p>
<p>I need people who understand the front end and dePython creation scenarios and help me create a functional and practical framework so that any developer does not waste hours typing and building HTML, CSS or even the basics of JS, and of course the system will be integrated with Bootstrap.</p>
<p>Code to create Html + elements  (Python code)<br></p><pre>from Framework import PyHtml<br>
from Framework import CssTool<p></p>
<p><span class="hashtag">#Template</span> for testes<br>
<span class="hashtag">#It</span>’s system work unique function with python tecnology<br>
<span class="hashtag">#Create</span> HTML and CSS files with less lines in your Body.<br>
<span class="hashtag">#You</span> no need create your web Project with hard dificult.<br>
<span class="hashtag">#Uses</span> .CreateNewHtml() &lt;–Where create one new HTML code.<br>
<span class="hashtag">#Its</span> very easy and cool.<br>
<span class="hashtag">#Powered</span> by Felipe Katao Tecnology.</p>
<p><span class="hashtag">#code</span> exemple:</p>
<p><span class="hashtag">#Class</span> for create<br>
html= PyHtml.HtmlPy()<br>
css= PyHtml.CssPy()<br>
resp=CssTool.Responsive()<br>
<span class="hashtag">#start</span> code</p>
<p>html.CreateNewHtml(“index2.html”,“index2.html”,1)</p>
<p>html.CreateMenuHtml(“index2.html”,"<span class="hashtag">#menu</span>",8)</p>
<p>resp.importCssResponsive(“index2.html”,3)<br>
resp.CardBootstrap(“index2.html”,50,“Hello”,“World friend”,“Loren inspum loren”,html.toNextLine(1,10))</p>
</pre>
<br>Result in html code:<br><pre>&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt; &lt;title&gt;index2.html&lt;/title&gt;
<p>&lt;link rel=‘stylesheet’ href=‘<a href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" class="inline-onebox-loading" rel="nofollow noopener">https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css</a>’ integrity=‘sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T’ crossorigin=‘anonymous’&gt;<br>
&lt;meta charset=‘UTF-8’&gt;<br>
&lt;meta name=‘viewport’content=‘width=device-width, initial-scale=1.0’&gt;<br>
&lt;link&gt;<br>
&lt;/head&gt;<br>
&lt;body&gt;<br>
&lt;nav id=’<span class="hashtag">#menu</span>’&gt;<br>
&lt;ul&gt;<br>
&lt;li&gt;Item&lt;/li&gt;</p>
<p>&lt;div class=‘card’ style=‘width: 10rem;’&gt;<br>
&lt;div class=‘card-body’&gt;<br>
&lt;h5 class=‘card-title’&gt;Hello&lt;/h5&gt;<br>
&lt;h6 class=‘card-subtitle mb-2 text-muted’&gt;World friend&lt;/h6&gt;<br>
&lt;p class=‘card-text’&gt;Loren inspum loren&lt;/p&gt;<br>
&lt;a href=’#’ class=‘card-link’&gt;link&lt;/a&gt;<br>
&lt;/div&gt;<br>
&lt;div&gt; &lt;li&gt;Item&lt;/li&gt;<br>
&lt;/ul&gt;<br>
&lt;/nav&gt;<br>
&lt;nav id=’<span class="hashtag">#menu2</span>’&gt;<br>
&lt;ul&gt;<br>
&lt;li&gt;Item&lt;/li&gt;<br>
&lt;li&gt;Item&lt;/li&gt;<br>
&lt;li&gt;Item&lt;/li&gt;<br>
&lt;/ul&gt;<br>
&lt;/nav&gt;<br>
&lt;div id=’<span class="hashtag">#hello</span>’&gt;Hello world&lt;/div&gt;</p>
<p>&lt;form action=’<span class="hashtag">#ok</span>’&gt;<br>
&lt;label&gt;Label&lt;/label&gt;<br>
&lt;input id=‘Input0’ type=‘text’ name=‘name’&gt;<br>
&lt;input type=‘submit’ value=‘save’&gt;<br>
&lt;/form&gt;<br>
&lt;/body&gt;<br>
&lt;/html&gt;</p></pre><br>
<br><br><br><p></p>
<p>Anyone who can help me or just take a look at the repository and leave a star there will be very happy, but whoever can really help me in this non-profit project would be much happier</p>
<p>Great friends!</p>
<p>Felipe Katão </p>
<p>Visit my github and folow me!  <a href="https://github.com/FelipeKatao" rel="nofollow noopener">Github Felipe Katao</a></p>

**Comment by ars1979** [comment, marked as answer]
<p>Try this,</p>
<p><a href="https://github.com/webyaml" rel="nofollow noopener">https://github.com/webyaml</a></p>
<p>You dont even need to be that great with python.</p>
